src/control/system/member_test.go

```diff
@@ -474,19 +474,19 @@ func TestSystem_Membership_CheckHostlist(t *testing.T) {
 			inHosts:         "foo-[1-3]",
 			expMissingHosts: "foo-[1-3]",
 		},
-		"ips cant resolve": {
+		"ips can't resolve": {
```

Comment:
@tanabarr is this going to cause an issue?  I can change it to something without a ' character if preferred but it's triggering our spell checker currently.

Reply:
that should be fine